README.md

```diff
@@ -1,26 +1,11 @@
-# Mapbox Vision SDK for Android
+# Mapbox Vision Android Examples
 
-The Vision SDK uses highly efficient neural networks to process imagery directly on user’s
-mobile or embedded devices, turning any connected camera into a second set of eyes
-for your car. In doing so, the Vision SDK enables the following user-facing features:
+This repository shows couple of Vision SDK usage examples:
+- Using video file as video source
+- Basic Vision AR functions
+- Basic Vision Safety functions
 
-- Augmented reality navigation with turn-by-turn directions
-- Classification and display of regulatory and warning signs
-- Object detection for vehicles, pedestrians, road signs, and traffic lights
-- Semantic segmentation of the roadway into 14 different classes (other, road, road_markup, flat_non_road, sky, building, car, cycle, person, road_markings_other, curb, double_yellow, traffic_sign, traffic_light)
-- Distance detection that indicates spacing to lead vehicle
-
-#### Hardware requirements
-
-VisionSDK requires Android 6 (API 23) and higher, with QC Snapdragon 650 // 710 // 8xx with Open CL support
-
-Some of devices that will work with VisionSDK:
-- Samsung Galaxy S8, S8+ // S9, S9+ // Note 8
-- Xiaomi Mi 6 // 8
-- HTC U11, U11+ // U12, U12+
-- OnePlus 5 // 6
-
-You can also check more details at [Vision SDK FAQ](https://vision.mapbox.com/#faq).
+For more information and frequently asked questions please visit [Vision SDK FAQ](https://vision.mapbox.com/#faq).
 
 ## Installation and setup
```

Comment:
I'd also alter text here since Examples is the only module remained

Reply:
OK.